web/src/buddy_list.ts

```diff
@@ -218,31 +234,13 @@ export class BuddyList extends BuddyListConf {
                                 {N: participant_count},
                             );
                         } else if (elem_id === "buddy-list-users-matching-view-section-heading") {
-                            if (participant_count) {
-                                tooltip_text = $t(
-                                    {
-                                        defaultMessage:
-                                            "{N, plural, one {# other subscriber} other {# other subscribers}}",
-                                    },
-                                    {N: total_human_subscribers_count - participant_count},
-                                );
-                            } else if (current_sub) {
-                                tooltip_text = $t(
-                                    {
-                                        defaultMessage:
-                                            "{N, plural, one {# subscriber} other {# subscribers}}",
-                                    },
-                                    {N: total_human_subscribers_count},
-                                );
-                            } else {
-                                tooltip_text = $t(
-                                    {
-                                        defaultMessage:
-                                            "{N, plural, one {# participant} other {# participants}}",
-                                    },
-                                    {N: total_human_subscribers_count},
-                                );
-                            }
+                            tooltip_text = $t(
+                                {
+                                    defaultMessage:
+                                        "{N, plural, one {# other subscriber} other {# other subscribers}}",
+                                },
+                                {N: non_participant_subscribers_count},
+                            );
```

Comment:
Keep the last `else` block that says "participants". For DMs this is the people in the DM, and the formal participants section is hidden.

I can see how this is confusing, and I was pondering if we should change how it works -- but the users matching narrow section is for all people who can see the narrow, and the first section is for users who have sent messages in that narrow (which isn't necessarily all people in a DM conversation audience).